Content.Server/Paper/PaperComponent.cs

```diff
@@ -10,6 +10,9 @@ public sealed class PaperComponent : SharedPaperComponent
         [DataField("content")]
         public string Content { get; set; } = "";
 
+        [DataField("useLocale")]
+        public bool UseLocale;
```

Comment:
is this even necessary? you're already doing a trygetstring for the locale in the code.

Reply:
Dunno if someone want to make book with contents looking like locale key but anyways that works like double check of intentions